build_tools/mako/compile_android_modules.sh

```diff
@@ -60,7 +58,7 @@ do
   extra_flags=()
   case "${target}" in
     "dylib-llvm-aot")
-      extra_flags+=('--iree-llvm-target-triple=aarch64-linux-android')
+      extra_flags+=('--iree-llvm-target-triple=aarch64-none-linux-android30')
```

Comment:
Just curious about the format of these triples... I've been using `android29` since that matches the version of the NDK I currently have downloaded. Does generic `android` work well enough, or are we going to have these sorts of pinned versions in several places?

Reply:
`aarch64` & `android30` are supposed to be driven from what ever we have here as `ANDROID_PLATFORM` & `ANDROID_ABI`
https://source.corp.google.com/piper///depot/google3/third_party/iree/build_tools/cmake/build_android.sh;l=50?q=build_android.sh&ss=piper%2FGoogle%2FPiper:google3%2Fthird_party%2Firee%2F
I used 30 as the most recent NDK have